src/AvTranscoder/transcoder/Transcoder.hpp

```diff
@@ -163,7 +163,7 @@ class AvExport Transcoder
     void addTranscodeStream(const InputStreamDesc& inputStreamDesc,
                             const ProfileLoader::Profile& profile, const float offset = 0);
 
-    void addDummyStream(const ProfileLoader::Profile& profile);
+    void addGeneratedStream(const ProfileLoader::Profile& profile);
```

Comment:
Sorry but once more, I am discussing the name of a method: here at first reading, it seems _addGeneratedStream_ means you "add a stream that has already been generated", while you would like to "add a stream that will be generated by the transcoder". So I'd prefer something like _addStreamGeneration_,  _addStreamToGenerated_ or _addStreamGenerator_, which seems clearer to me. Don't you think?